sdk/formrecognizer/azure-ai-formrecognizer/azure/ai/formrecognizer/_models.py

```diff
@@ -231,70 +232,114 @@ class FormContentType(str, Enum, metaclass=CaseInsensitiveEnumMeta):
 class Point(namedtuple("Point", "x y")):
     """The x, y coordinate of a point on a bounding box or polygon.
 
-    :ivar float x: x-coordinate
-    :ivar float y: y-coordinate
-
     .. versionadded:: v2.1
         Support for *to_dict* and *from_dict* methods
     """
+    x: float
+    """x-coordinate"""
+    y: float
+    """y-coordinate"""
+
 
     __slots__ = ()
 
-    def __new__(cls, x, y):
+    def __new__(cls, x: float, y: float) -> "Point":
         return super().__new__(cls, x, y)
 
-    def to_dict(self) -> dict:
-        """Returns a dict representation of Point.
-
-        :return: dict
-        :rtype: dict
-        """
+    def to_dict(self) -> Dict:
+        """Returns a dict representation of Point."""
         return {"x": self.x, "y": self.y}
 
     @classmethod
-    def from_dict(cls, data: dict) -> "Point":
+    def from_dict(cls, data: Dict) -> "Point":
         """Converts a dict in the shape of a Point to the model itself.
 
-        :param dict data: A dictionary in the shape of Point.
+        :param Dict data: A dictionary in the shape of Point.
         :return: Point
         :rtype: Point
         """
         return cls(x=data.get("x", None), y=data.get("y", None))
 
 
+class TextAppearance:
+    """An object representing the appearance of the text line.
+
+    .. versionadded:: v2.1
+        Support for *to_dict* and *from_dict* methods
+    """
+    style_name: str
+    """The text line style name.
+        Possible values include: "other", "handwriting"."""
```

Comment:
I noticed when I generated documentation that the `:` did a weird thing and created a `Type` in the docstring. Not sure if you'll see the same thing when you run sphinx but I just removed the `:` and replaced with `possible values are...`

Reply:
Gave the docs a skim and didnt see this causing a problem, however I'll do a more in-depth check, tracking issue: https://github.com/Azure/azure-sdk-for-python/issues/28326